pkg/bisect/bisect.go

```diff
@@ -241,8 +242,18 @@ func (env *env) bisect() (*Result, error) {
 	}
 
 	cfg := env.cfg
-	if err := build.Clean(cfg.Manager.TargetOS, cfg.Manager.TargetVMArch,
-		cfg.Manager.Type, cfg.Manager.KernelSrc); err != nil {
+	buildCfg := &instance.BuildKernelConfig{
```

Comment:
I wonder if we can reduce duplication here as it's essentially the same object as the `instance.BuildKernelConfig` that we create in `build()`. AFAIK only `CompilerBin` is different?

Reply:
Is the current state close to what you had in mind ? It seems a bit messy to me but maybe with further refactorings or better name we can make it look a bit better

Reply:
Hmm, no, not exactly. I had a smaller change in mind, local to `bisec.go` only :)

E.g. have some base `instance.BuildKernelConfig` field in `env` and only patch the `Compiler` field in `build()` because the rest are absolutely the same.

Reply:
Ah right, I totally misunderstood you sorry ahah. I'll get back to it on monday

Reply:
*(totally on monday)* I implemented what you suggested but actually there was more than the `CompilerBin` which was different, also the kernel config, sysctl file, cmdline, and userspace directory. Let me know if you prefer getting back to the previous state instead ;)

Reply:
> sysctl file, cmdline, and userspace directory.

Are these not taken from the same object? `kern` is `env.cfg.Kernel` -- the exact same one as in `runImpl()`? 

`KernelConfig` should be taken from `bisectEnv` indeed.

Reply:
Indeed, that seems to work :)